academic-services/project-academic-knowledge/reference-extended-removal.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Deprecation of Extended Entity Attribute
+description: Information about the deprecation of the Extended Entity Attribute and guidance for migration
+ms.topic: reference
+ms.date: 2020-07-29
+---
+
+# Deprecation of Entity Extended Attributes
+
+The Extended Entity Attributes is deprecated and legacy application support will end on December 21, 2020. Evaluate and Histogram API will return "attribute not found error" if Extended Entity Attributes is a part of the request. The Extended Entity Attributes provides extended entity information such as paper BibTex venue name, citation contexts, original paper title, and DOI. You can migrate your application to leverage other attributes that provides equivalent information. The other equivalent attributes can also perform more operations. Please transition your API usage to leverage the equivalent available attributes.
```

Comment:
>Evaluate and Histogram API will return "attribute not found error" if Extended Entity Attributes is a part of the request. The Extended Entity Attributes provides extended entity information such as paper BibTex venue name, citation contexts, original paper title, and DOI. You can migrate your application to leverage other attributes that provides equivalent information. The other equivalent attributes can also perform more operations. Please transition your API usage to leverage the equivalent available attributes. [](start = 107, length = 521)

I think we can simplify this to:

"This means that the Extended attribute will no longer be able to be requested from the API, and doing so will result in an "attribute not found" error.

All sub-attributes previously available as part of the Extended attribute have been moved to the top-level of the schema with the following mapping:

Friendly name | Extended sub-attribute | New top-level attribute
--- | --- | ---
Display name | E.DN | DN
...

"